seed_scheduler/settings.py

```diff
@@ -188,6 +188,9 @@
     'seed_scheduler.scheduler.tasks.queue_tasks': {
         'queue': 'priority',
     },
+    'seed_scheduler.scheduler.tasks.requeue_failed_tasks': {
+        'queue': 'priority',
+    },
```

Comment:
I like the idea of having a separate queue as then an increase in failures over time is easier to spot.